microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -301,6 +301,12 @@ def _extract_test_results(
         for subtest in subtests:
             status = subtest_status[subtest]
             message = messages.get(status, "")
+
+            if status == TestStatus.RUNNING:
+                # Sub-test started running but didn't finish within the stipulated time.
+                # It should be treated as a failure.
+                status = TestStatus.FAILED
```

Comment:
should we extend the running time?

Reply:
No, the running time is already very generous, and is not the root cause. There is some underlying issue in the kernel or Cloud Hypervisor that is causing this. The goal of this PR is to simply report the status correctly.